soda/core/soda/sodacl/sodacl_parser.py

```diff
@@ -1450,6 +1425,9 @@ def antlr_parse_column_configuration(self, text: str) -> AntlrParser:
     def antlr_parse_threshold(self, text: str) -> AntlrParser:
         return AntlrParser(text, lambda p: p.threshold())
 
+    def get_data_source_scan_cfgs(self):
+        return self.sodacl_cfg._get_or_create_data_source_scan_cfgs(self.data_source_name)
```

Comment:
Can you also change the function name in SodaCLCfg and remove `_` ? We use it outside the class, so it doesn't make sense there.

Reply:
done